### PR TITLE
Standarize existing sizes from 1024 to 1000 bytes

### PR DESCRIPTION
**Pull request by spetz**

### CODE REVIEW

File: bench/src/benchmark_result.rs [hunk 1]
```diff
@@ -90,8 +90,8 @@ impl BenchmarkResults {
             .as_secs_f64()
             * 1000.0;
         let average_throughput =
-            total_size_bytes as f64 / total_duration / 1024.0 / 1024.0 / self.results.len() as f64;
-        let total_throughput = total_size_bytes as f64 / total_duration / 1024.0 / 1024.0;
+            total_size_bytes as f64 / total_duration / 1000.0 / 1000.0 / self.results.len() as f64;
+        let total_throughput = total_size_bytes as f64 / total_duration / 1000.0 / 1000.0;
```
Comment (numinnex) [inline bench/src/benchmark_result.rs]: I propose this instead - `total_size_bytes as f64 / total_duration / 1e6 / self.results.len()`

File: bench/src/consumer.rs [hunk 2]
```diff
@@ -107,7 +107,7 @@ impl Consumer {
         let end_timestamp = Instant::now();
         let duration = end_timestamp - start_timestamp;
         let average_latency: Duration = latencies.iter().sum::<Duration>() / latencies.len() as u32;
-        let average_throughput = total_size_bytes as f64 / duration.as_secs_f64() / 1024.0 / 1024.0;
+        let average_throughput = total_size_bytes as f64 / duration.as_secs_f64() / 1000.0 / 1000.0;
```
Comment (numinnex) [inline bench/src/consumer.rs]: same as above

File: bench/src/producer.rs [hunk 3]
```diff
@@ -82,7 +82,7 @@ impl Producer {
         let duration = end_timestamp - start_timestamp;
         let average_latency: Duration = latencies.iter().sum::<Duration>() / latencies.len() as u32;
         let total_size_bytes = total_messages * self.message_size as u64;
-        let average_throughput = total_size_bytes as f64 / duration.as_secs_f64() / 1024.0 / 1024.0;
+        let average_throughput = total_size_bytes as f64 / duration.as_secs_f64() / 1000.0 / 1000.0;
```
Comment (numinnex) [inline bench/src/producer.rs]: same use SI notation

File: bench/src/benchmark_result.rs (hunk 1 above)
Comment (spetz) [reply bench/src/benchmark_result.rs]: Done

File: bench/src/consumer.rs (hunk 2 above)
Comment (spetz) [reply bench/src/consumer.rs]: Done

File: bench/src/producer.rs (hunk 3 above)
Comment (spetz) [reply bench/src/producer.rs]: Done

**Review by numinnex — APPROVED** [review]

**Pull request merged**